src/utilities/constants.ts

```diff
@@ -26,3 +26,6 @@ export const APP_NAME = {
   PortfolioItem: 'catalog',
   ServiceInventory: 'topology'
 };
+
+export const BEFORE_TYPE = 'before';
+export const AFTER_TYPE = 'after';
```

Comment:
These constant names do not match the `TYPE_BEFORE` and `TYPE_AFTER` imported in the schema.